src/cljs/athens/electron/db_menu/db_list_item.cljs

```diff
@@ -59,10 +61,21 @@
 
 (defn db-list-item
   [{:keys [db is-current]}]
-  [:div (use-style db-list-item-style)
-   (if is-current
-     [:div.body.is-current
-      [db-list-item-content {:db db}]]
-     [:button.body.button {:onClick
-                           #(dispatch [:db-picker/select-db db])}
-      [db-list-item-content {:db db}]])])
+  (let [state                   (r/atom {:show-remove-button false})
+        mouse-over-handler      #(swap! state assoc :show-remove-button true)
+        mouse-leave-handler     #(swap! state assoc :show-remove-button false)
+        remove-db-click-handler (fn [e]
+                                  (dispatch [:db-picker/remove-db db])
+                                  (.. e stopPropagation))]
+    (fn []
+      [:div (use-style db-list-item-style)
+       (if is-current
+         [:div.body.is-current
+          [db-list-item-content {:db db}]]
+         [:button.body.button {:onClick      #(dispatch [:db-picker/select-db db])
+                               :onMouseOver  mouse-over-handler
```

Comment:
Why do we want to show UI element when cursor enters the area?
Sounds like "surprise"